packages/wallet/backend/src/incomingPayment/service.ts

```diff
@@ -16,11 +18,7 @@ interface IIncomingPaymentService {
   ) => Promise<Transaction>
   getPaymentDetailsByUrl: (url: string) => Promise<PaymentDetails>
   createIncomingPaymentTransactions: (
-    paymentPointerId: string,
-    amount: bigint | null,
-    asset: Asset,
-    description?: string,
-    expiresAt?: string
+    params: CreateIncomingPaymentParams
```

Comment:
We should choose if we use `params` or `args`.

Reply:
for now we used Params the most, and I'd continue with params, since args is what the generated graphql uses

Reply:
will change what I encounter as args to params aswell from now on